docs/readme-tw-caldav.md

```diff
@@ -29,7 +29,8 @@ TW <-> Caldav will make the following mappings between items:
   - `L` <-> 9
   - `M` <-> 5
   - `H` <-> 1
-- TW `annotations`, `uuid` <-> `DESCRIPTION`
+- TW `annotations` <-> `DESCRIPTION` (one annotation <-> one line in description)
+- TW `uuid`, `uuid` <-> `X-SYNCALL-TW-UUID`
```

Comment:
Is it intentional to repeat the `uuid` ?

Reply:
My bad. It's a typo in the doc. The implementation just has one uuid in X-SYNCALL-TW-UUID